app/javascript/application.js

```
@@ -0,0 +1,22 @@
+import { Turbo } from '@hotwired/turbo-rails';
+Turbo.start();
+
+import Vue from 'vue/dist/vue.esm';
+import VueResource from 'vue-resource';
+import axios from 'axios';
```

ESLint

[error] 'axios' is defined but never used.

(no-unused-vars)